ubuntu_image/__main__.py

```diff
@@ -158,15 +186,106 @@ def parseargs(argv=None):
         default=False, action='store_true',
         help=_("""Continue the state machine from the previously saved state.
         It is an error if there is no previous state."""))
+    return subcommand
+
+
+def parseargs(argv=None):
+    parser = argparse.ArgumentParser(
+        prog=PROGRAM,
+        description=_('Generate a bootable disk image.'),
+        formatter_class=SimpleHelpFormatter)
+
+    parser.add_argument(
+        '--version', action='version',
+        version='{} {}'.format(PROGRAM, __version__))
```

Comment:
I think we need to somehow add the --version argument to the common arguments, since right now it is not recognizable. The tests are failing because of that as well.

Reply:
Interesting, I didn't notice when running coverage test suites.
Thanks for the mention.